arkdep-build.d/manjaro-gnome-ge-ogui/overlay/post_bootstrap/etc/dconf/db/local.d/01-manjaro-theming

```diff
@@ -0,0 +1,16 @@
+[org/gnome/desktop/background]
+color-shading-type='solid'
+picture-options='zoom'
+picture-uri='file:///usr/share/backgrounds/manjaro/abstract-l.png'
+picture-uri-dark='file:///usr/share/backgrounds/manjaro/abstract-d.png'
+primary-color='#3465a4'
+secondary-color='#000000'
+
+[org/gnome/desktop/interface]
+accent-color='maia'
```

Comment:
When this works for GNOME 46 and gets ignored, fine with me. Everything else was already tested by me and is good to go.

Reply:
I validated the DB still builds fine under GNOME 46.